src/ripple/app/misc/ValidatorList.h

```diff
@@ -377,7 +377,8 @@ ValidatorList::onConsensusStart (
                 list.second.expiration <=
                 timeKeeper_.now().time_since_epoch().count())
             removePublisherList (list.first);
-        else if (! list.second.available)
```

Comment:
Are all published lists required to have an expiration?

Reply:
Yes
https://github.com/ripple/rippled/blob/3e5490ef6db9cbc3abdff04655d80952b288d097/src/ripple/app/misc/impl/ValidatorList.cpp#L302-L313